Replicator/Puller.cc

```diff
@@ -39,13 +39,15 @@ namespace litecore { namespace repl {
 
     Puller::Puller(Replicator *replicator)
     :Delegate(replicator, "Pull")
+#if __APPLE__
+    // This field must go before _revFinder because "this" is passed in "new RevFinder(replicator, this)," which will
```

Comment:
FYI, this order actually doesn't matter; I looked it up to make sure:

> The order of member initializers in the list is irrelevant  ... non-static data members are initialized in order of declaration in the class definition

So it's the order in the header that matters. But it's a good idea to have the same ordering in the implementation to avoid confusion!

This does mean that your explanatory comment should actually go in the declaration in the header instead.

Reply:
Accepted.